### PR TITLE
Fix regexes and adjust tests for correct Discord mentions syntax

### PR DESCRIPTION
**Pull request by jingjing142**
This PR solves this issue: 
- points.js commands regexes need to pass their tests https://github.com/TheOdinProject/odin-bot-v2/issues/92

After discussing it with Tatiana, I changed @user and @odin-bot to <@!1233456679> in the test file to align it with Discord's syntax for mentions and corrected some minor errors. That's also why the snapshots have changed. 

Other than that, I just adjusted the regexes to make them pass the tests. 

I think my Prettier replaced the double quotes by single quotes and removed unnecessary semi-colons and whitespace... Hope that's ok.

### CODE REVIEW

File: botCommands/__snapshots__/points.test.js.snap [hunk 1]
```diff
@@ -55,36 +55,36 @@ exports[`/leaderboard callback returns correct output 8`] = `
...
+exports[`<@!1233456679> ++ callback returns correct output for up to five mentioned users 1`] = `"Woot! <@2> now has 34 points"`;
 
-exports[`@user ++ callback returns correct output for up to five mentioned users 2`] = `"Sweet! <@2> now has 22 points"`;
+exports[`<@!1233456679> ++ callback returns correct output for up to five mentioned users 2`] = `"Sweet! <@2> now has 22 points"`;
 
-exports[`@user ++ callback returns correct output for up to five mentioned users 3`] = `"Nice! <@2> now has 3 points"`;
+exports[`<@!1233456679> ++ callback returns correct output for up to five mentioned users 3`] = `"Nice! <@2> now has 3 points"`;
 
-exports[`@user ++ callback returns correct output for up to five mentioned users 4`] = `"Nice! <@2> now has 1 point"`;
+exports[`<@!1233456679> ++ callback returns correct output for up to five mentioned users 4`] = `"Nice! <@2> now has 1 point"`;
```
Comment (twalton83) [inline botCommands/__snapshots__/points.test.js.snap:88]: Thanks for catching this and ensuring our snapshot tests were correct.

File: botCommands/points.test.js [hunk 2]
```diff
@@ -1,79 +1,65 @@
...
-        },
-        toString : () => `<@${id}>`
-      }
-    })
-  }
-})
+const axios = require('axios');
+const {
+  Guild, Channel, Client, User,
+} = require('discord.js');
```
Comment (twalton83) [inline botCommands/points.test.js:4]: I think this looks a bit off, this was from ESLint?

**Review by twalton83 — COMMENTED** [review]
Looks good to me thus far, someone with write access will need to review and merge.

File: botCommands/__snapshots__/points.test.js.snap (hunk 1 above)
Comment (twalton83) [reply botCommands/__snapshots__/points.test.js.snap:88]: It's small, but I want to change "<@!1233456679>" to something a little more uniform, like <@!123456789>, just so it's a little more predictable. My silly string was just an example. Go ahead and change it then update the snapshot tests one more time, then I will merge.

Comment (jingjing142) [reply botCommands/__snapshots__/points.test.js.snap:88]: Ok, no worries. Will do it later today :)

Comment (jingjing142) [reply botCommands/__snapshots__/points.test.js.snap:120]: Changed it :)

File: botCommands/points.test.js (hunk 2 above)
Comment (jingjing142) [reply botCommands/points.test.js:4]: Yes, I didn't touch that part of the code after running eslint autofix. Also disabled eslint and prettier before changing the example string above.

File: botCommands/__snapshots__/points.test.js.snap (hunk 1 above)
Comment (twalton83) [reply botCommands/__snapshots__/points.test.js.snap:120]: Were all of the snapshots updated? In the preview it looks like some still use the old string.

Comment (jingjing142) [reply botCommands/__snapshots__/points.test.js.snap:120]: Oh whoopsie! Fixing it now

Comment (jingjing142) [reply botCommands/__snapshots__/points.test.js.snap]: Fixed it! Please take a look :)

**Comment by twalton83** [comment]
Just messaged you on Discord re: the updated snapshots

**Comment by twalton83** [comment]
Also, do me a favor and share a screenshot of the passing tests :)

**Comment by jingjing142** [comment]
> Also, do me a favor and share a screenshot of the passing tests :)

Sure, here you go :)

image

**Review by twalton83 — APPROVED** [review]
Alrighty, thanks for updating those snapshot tests and fixing this!

**Pull request merged**